core/app/models/spree/shipment_handler.rb

```
@@ -36,5 +37,23 @@ def update_order_shipment_state
       order.update_columns(shipment_state: new_state, updated_at: Time.current)
     end
+
+    def track_package_shipped_event
+      order = @shipment.order
+
+      analytics_event_handlers = Spree::Analytics.event_handlers.map do |handler|
+        handler.new(user: order.user, session: nil, request: nil, store: order.store)
+      end
+
+      analytics_event_handlers.each do |handler|
+        handler.handle_event('package_shipped', {order: order, shipment: @shipment})
```

RuboCop (1.73)

[convention] Space inside { missing.

(Layout/SpaceInsideHashLiteralBraces)